docs/providers.rst

```diff
@@ -153,7 +153,7 @@ Add the following configuration to your settings:
                 "key": "MEMAPPIDPREFIX",
 
                 # The certificate you downloaded when generating the key.
-                "certificate_key": """-----BEGIN PRIVATE KEY-----
+                "certificate": """-----BEGIN PRIVATE KEY-----
```

Comment:
This is intentionally changed to `certificate_key` from the looks of it > https://github.com/pennersr/django-allauth/commit/e43bfd96039eea009925888ab8e4efb8428a69e1#